pkg/ingress/config/ingress_config.go

```diff
@@ -676,6 +688,44 @@ func (m *IngressConfig) convertWasmPlugin([]common.WrapperConfig) []config.Confi
 			Spec: wasmPlugin,
 		})
 	}
+	// add wasm plugin from nacos for mcp server
+	if m.RegistryReconciler != nil {
+		rulesFromMcp := m.RegistryReconciler.GetAllConfigs()[gvk.WasmPlugin.String()]
+		if len(rulesFromMcp) > 0 {
```

Comment:
The processing logic here is not consistent with VirtualService. The config obtained by m.RegistryReconciler.GetAllConfigs()[gvk.VirtualService.String()] has already been processed, but WasmPlugin has not. What was the consideration behind this?